modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/util/JSONObjectDefinitionTestUtil.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.web.internal.portlet.action.test.util;
+
+import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONObject;
+
+import java.util.Objects;
+
+/**
+ * @author Eva Budai
+ */
+public class JSONObjectDefinitionTestUtil {
+
+	public static JSONObject getAuthorJSONObject(JSONObject jsonObject) {
+		return getObjectFieldJSONObject(jsonObject, "creator");
+	}
+
+	public static JSONObject getCreateDateJSONObject(JSONObject jsonObject) {
+		return getObjectFieldJSONObject(jsonObject, "createDate");
+	}
+
+	public static JSONObject getExternalReferenceCodeJSONObject(
+		JSONObject jsonObject) {
+
+		return getObjectFieldJSONObject(jsonObject, "externalReferenceCode");
+	}
+
+	public static JSONObject getIDJSONObject(JSONObject jsonObject) {
+		return getObjectFieldJSONObject(jsonObject, "id");
+	}
+
+	public static JSONObject getModifiedDateJSONObject(JSONObject jsonObject) {
+		return getObjectFieldJSONObject(jsonObject, "modifiedDate");
+	}
+
+	public static JSONObject getObjectFieldJSONObject(
+		JSONObject jsonObject, String objectField) {
+
+		JSONArray jsonArray = jsonObject.getJSONArray("objectFields");
+
+		for (int i = 0; i < jsonArray.length(); i++) {
+			JSONObject jsonObject1 = (JSONObject)jsonArray.get(i);
+
+			if (Objects.equals(jsonObject1.getString("name"), objectField)) {
+				return jsonObject1;
+			}
+		}
+
+		return null;
+	}
+
+	public static JSONObject getStatusDateJSONObject(JSONObject jsonObject) {
```

Comment:
I think the name of this method is wrong? Shouldn't it be getStatus ? Also if its as simple as calling the `getObjectFieldJSONObject` we shouldn't have a dedicated method.
But i don't think this should work since `status` is a field that is not inside `objectFields` :thinking: